test/e2e/util.go

```diff
@@ -1116,10 +1088,6 @@ func createGenericDeploymentAW(context *context, name string) *arbv1.AppWrapper
 			AggrResources: arbv1.AppWrapperResourceList{
 				GenericItems: []arbv1.AppWrapperGenericResource{
 					{
-						ObjectMeta: metav1.ObjectMeta{
```

Comment:
@astefanutti can you please share why `ObjectMeta` field was removed?

Reply:
Generally `metav1.TypeMeta` and  `metav1.ListMeta` are used on root structs, that's used so the REST Mapper can map  the APIs to CRDs. I've removed those on the `AppWrapperResourceList` struct as it's not a root struct, that does not correspond to any resource. These are not used by the MCAD controller, and lots of test samples do not declare them. It seems to me those present have been copied from the old non-generic items.